stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -71,6 +72,13 @@ class PaymentSheet internal constructor(
          */
         var googlePay: GooglePayConfiguration? = null,
 
+        /**
+         * The color of the Pay or Add button. Keep in mind the text color is white.
+         *
+         * If set, PaymentSheet displays the button with this color.
+         */
+        var primaryButtonColor: ColorStateList? = null,
```

Comment:
maybe renamed to buttonBackgroundColor and clarify in the java doc? 
Also could you attach a screenshots when this is overridden?

Reply:
we collectively agreed to keep the name